packages/cli/commands/sandbox/delete.js

```diff
@@ -117,6 +117,9 @@ exports.handler = async options => {
     }
     process.exit(EXIT_CODES.SUCCESS);
   } catch (err) {
+    logger.debug(i18n(`${i18nKey}.debug.error`));
+    logger.debug(err.error);
```

Comment:
we do have a `logErrorInstance` helper function. I forget exactly how it works, but it may be worth seeing if it can work here too

Reply:
Good call! Updating those now